README.md

```diff
@@ -0,0 +1,71 @@
+# Core-CMS-Custom
+
+Extensions of the [Core CMS] project
+
+## Related Repositories
+
+- [Camino], a Docker container-based deployment scheme
+- [Core Portal], the base Portal code for TACC WMA CMS Websites
+- [Core CMS Resources], the original solution for extensions of the [Core CMS] project
+- [Core Portal Deployments], private repository that facilitates deployments of [Core Portal] images via [Camino] and Jenkins
+
+## Architecture
+
+- [`./src/apps`](./src/apps/): Contains any additional Django applications
+- [`./src/taccsite_cms`](./src/taccsite_cms/): Contains settings files which specify additional apps, static files and middleware to load on top of Core CMS, along with standard Core CMS settings files
+- [`./src/taccsite_custom`](./src/taccsite_custom/): Contains static assets and templates, organized in the way that Django CMS expects them before imported via `python manage.py collectstatic`.
+
+## Local Development Setup
+
+### Prequisites for Running the CMS
+
+See [Core-CMS](https://github.com/TACC/Core-CMS#prequisites-for-running-the-cms).
+
+### Code Configuration
+
+#### Settings
+
+See [Core-CMS](https://github.com/TACC/Core-CMS#settings).
+
+## Running the CMS
+
+### Running in Development Mode
+
+A `Makefile` has been included for convenience in each CMS project. You may use:
+
+```bash
+cd custom-project-dir
+make start
+```
+
+The `Makefile` commands assume shared CMS databases. To use a unique database:
+
+```bash
+cd custom-project-dir
+docker-compose -f docker-compose.dev.unique.yml build
+docker-compose -f docker-compose.dev.unique.yml up
```

Comment:
```suggestion
```

Drop everything for https://github.com/TACC/Core-CMS-Custom/pull/10, so PR is not dependent.